src/main/java/org/generationcp/middleware/api/study/StudyDTO.java

```diff
@@ -3,6 +3,7 @@
 import org.pojomatic.Pojomatic;
 import org.pojomatic.annotations.AutoProperty;
 
+@Deprecated
```

Comment:
@lik-83 if I am not wrong, we have said we will replace this class by the new one wherever it is used in a new ticket, right?